### ISSUE TITLE
how does one add the show year to the rename process?

### ISSUE THREAD

**Issue by nydave69** [pymedusa/Medusa#10156]
**Describe the bug**

how does one add the show year to the rename process?
Only thing date wise i can find is Post-Processing Date: not what I need.

There are so many show remakes I need it to keep them a part.
I'm manually doing this a number of times a day.

Any help would be appreciated.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Medusa (please complete the following information):**
 - OS: [e.g. ubuntu 14.04 x64]
 - Branch: [e.g. master, develop]
 - Commit: [e.g. c65ca2]
 - Python version: [e.g. 3.6.8]
 - Database version: [e.g. 44.12]

**Debug logs (at least 50 lines):**
General > Advanced Settings > Enable debug
<details>

```
<-- Please replace this whole line with your debug logs -->
```
</details>

**Additional context**

**Comment by gleepwurp** [comment]
As far as I know, the rename process only handles the episodes, and not the show's parent folder.

However, in the Manage --> General option, about halfway down, you have a "Append (year) to each show title" option you can enable.  Don't think it will rename the existing shows you have, but will ensure that any new shows added should have the year in the name.

By extension, if your show now has the year in its title, the episodes should get the year too, if using the show's name in your episodes post-processing naming options.

**Comment by nydave69** [comment]
> However, in the Manage --> General option, about halfway down, you have a "Append (year) to each show title" option you can enable. Don't think it will rename the existing shows you have, but will ensure that any new shows added should have the year in the name.

I have that setting and it does create the new series folder with the year.   halfway there...

But it doesn't hold true for the episodes..  
The year is not considered part of the show name so the episodes don't get the year added..  wish it was would also solve this problem

Without the show year I cannot let the post-processor move the file to the final destination since that could be the wrong series and destroy a good series episode.

I have to manually move it. 

Just curious how others deal with all the reboots, sequels, remakes..

**Comment by p0psicles** [comment]
So if I understand correct. What you'd like to have is when you've enabled the "add year" option. That when pp/renaming an episode, it uses the show name, inclusive the year, for the episode name?

**Comment by medariox** [comment]
<html>
<body>


Air Date: | %M | 12
-- | -- | --
  | %D | 21
  | %Y | 2021
Post-Processing Date: | %CM | 12
  | %CD | 21
  | %CY | 2021


</body>
</html>

You can use the air date in the episode naming settings.

**Comment by nydave69** [comment]
> So if I understand correct. What you'd like to have is when you've enabled the "add year" option. That when pp/renaming an episode, it uses the show name, inclusive the year, for the episode name?

Yes so that renames would always start with  ie,  Show Name (2018)

**Comment by nydave69** [comment]
> You can use the air date in the episode naming settings.

No that doesn't work, 
 it's not the show year,   it changes in a series season as it lapse years

**Comment by medariox** [comment]
This should work: https://github.com/pymedusa/Medusa/pull/10183
